lib/player.rb

```diff
@@ -1,9 +1,15 @@
 class Player
 
-  def turn(display, board, mark)
-    display.choose_cell
-    number = display.receive_integer
-    board.mark_board(number, mark)
-    display.present_board_with_squares(board)
+  def initialize(board, display, mark)
+    @board = board
+    @display = display
+    @mark = mark
+  end
+
+  def turn
+    @display.choose_cell
+    number = get_cell
+    @board.mark_board(number, @mark)
+    @display.present_board_with_squares(@board)
```

Comment:
Does the board get printed each turn? Could potentially be called from your runner (I think this is your game class?)